src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form form--big"
+      action="search"
+      data-qa="big"
+    >
+      <input
+        class="input form--big__input"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form form--small"
+      action="search"
+      data-qa="small"
+    >
+      <input
+        class="input form--small__input"
+        type="text"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
The class name 'form--small' suggests a modifier in BEM methodology, but there is no corresponding base block. Ensure that class names represent the meaning of the content and not the styles or tag names.